resources/xrp.js

```diff
@@ -35,10 +50,11 @@ window.onload = () => {
             })
             .then(() => {
                 alert("Configuration Updated. Please reset the XRP");
-            })
+            });
         }
         catch (e) {
-            alert("Invalid JSON. Please check and try again");
+            console.log("Invalid JSON. Please check and try again. Error: " + e);
+            alert("Invalid JSON. Please check and try again.\n" + e);
```

Comment:
should be `e.message`

Reply:
Fair. We don't need to show the error type. Will change.